ush/forecast_postdet.sh

```diff
@@ -439,11 +440,11 @@ EOF
   fi
 
   # Stochastic Physics Options
-  if [[ ${SET_STP_SEED:-"YES"} = "YES" ]]; then
+  if [[ ${DO_SPPT:-"NO"} = "YES" ]]; then
     ISEED_SKEB=$((current_cycle*1000 + MEMBER*10 + 1))
     ISEED_SHUM=$((current_cycle*1000 + MEMBER*10 + 2))
-    ISEED_SPPT=$((current_cycle*1000 + MEMBER*10 + 3))
-    ISEED_CA=$(( (current_cycle*1000 + MEMBER*10 + 4) % 2147483647 ))
+    ISEED_SPPT=$((current_cycle*10000 + ${MEMBER#0}*100 + 3)),$((current_cycle*10000 + ${MEMBER#0}*100 + 4)),$((current_cycle*10000 + ${MEMBER#0}*100 + 5)),$((current_cycle*10000 + ${MEMBER#0}*100 + 6)),$((current_cycle*10000 + ${MEMBER#0}*100 + 7))
```

Comment:
Is there any reason we don't change ISEED_SKEB as we do for SPPT?

Reply:
@bingfu-NOAA The goal is to emulate your workflow. Should ISEED_SKEB be the same as ISEED_SPPT?

Reply:
ideally, we need to set all the ISEED following the same algorithm and style.

Reply:
uploaded

Reply:
suggested revisions. We should not use DO_SPPT to control seeds setting for others.   
if [[ ${SET_STP_SEED:-"YES"} = "YES" ]]; then
    ISEED_SKEB=$((current_cycle*10000 + ${MEMBER#0}*100 + 3)),$((current_cycle*10000 + ${MEMBER#0}*100 + 4)),$((current_cycle*10000 + ${MEMBER#0}*100 + 5)),$((current_cycle*10000 + ${MEMBER#0}*100 + 6)),$((current_cycle*10000 + ${MEMBER#0}*100 + 7))
    ISEED_SHUM=$((current_cycle*10000 + MEMBER#0*100 + 2))
    ISEED_SPPT=$((current_cycle*10000 + ${MEMBER#0}*100 + 3)),$((current_cycle*10000 + ${MEMBER#0}*100 + 4)),$((current_cycle*10000 + ${MEMBER#0}*100 + 5)),$((current_cycle*10000 + ${MEMBER#0}*100 + 6)),$((current_cycle*10000 + ${MEMBER#0}*100 + 7))
    ISEED_CA=$(( (current_cycle*10000 + ${MEMBER#0}*100 + 18) % 2147483647 ))
    ISEED_LNDP=$(( (current_cycle*10000 + MEMBER#0*100 + 5) % 2147483647 ))
  else